api/v1/credential_types.go

```diff
@@ -11,6 +12,15 @@ import (
 type CredentialSpec struct {
```

Comment:
Too bad we have to repeat all these types here. Hopefully when we switch the API to use the generated proto stubs we can reuse those here as well.